src/current/_includes/v23.2/known-limitations/physical-cluster-replication.md

```diff
@@ -0,0 +1,14 @@
+- Physical cluster replication is supported only on CockroachDB {{ site.data.products.core }} in new v23.2 clusters. That is, clusters that have been upgraded from a previous version of CockroachDB will not support physical cluster replication.
+- Cockroach Labs has tested physical cluster replication at the following scale:
```

Comment:
I'll let @amruss @stevendanna weigh in on the framing. I think either works, i.e. "tested" or "supported", so happy to adjust. 

The contact support part: this feature is in Preview, and every preview note comes with that link to contact support. I don't know if this is overly encouraging that — is it useful to hear from users who are considering this feature at scale for our development purposes?

Reply:
Gotcha. Yeah, that one's tough. It's absolutely useful to hear from them, and we should definitely be consistent across our docs. Mostly what I want is to really set expectations that if they try a scale higher than this, it may not work, and I don't want that to come back on us. Make sense? What do you think? Maybe there's another way of being very clear about that? (But also gentle. Huh, docs are hard :) )